GUIs/ScriptRunnerPyQt5_GUI/ScriptGUIrunner.py

```diff
@@ -603,7 +587,29 @@ def confirmation(com):
 
 
 if __name__ == "__main__":
+    QApplication.setAttribute(QtCore.Qt.AA_EnableHighDpiScaling, True)
+    QApplication.setAttribute(QtCore.Qt.AA_UseHighDpiPixmaps, True)
```

Comment:
Codacy found an issue: ['QApplication' may be undefined, or defined from star imports: PyUIs.firstconf, scripFunc.scripLINUXONLY, scripFunc.scripUNIMULTI, scripFunc.scripWINONLY, threading (F405)](https://app.codacy.com/manual/vipersniper0501/CP_Scripts2/pullRequest?prid=5927260)